numpy/core/shape_base.py

```diff
@@ -183,23 +183,25 @@ def vstack(tup):
     """
     Stack arrays in sequence vertically (row wise).
 
-    Take a sequence of arrays and stack them vertically to make a single
-    array. Rebuild arrays divided by `vsplit`.
+    This is equivalent to concatenation along the first axis after 1d arrays of
```

Comment:
I think we settled on the form `1-D` for these things, although a grep shows some leftovers.